docs/modules/ROOT/generate-nav.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+readonly WORKING_DIR="$(pwd)/pages"
+readonly NEW_NAV_FILE_NAME="nav.adoc"
+readonly OUTPUT_NAV_LOCATION="$(pwd)/$NEW_NAV_FILE_NAME"
+readonly PARENT_DIR="/home/alanroth/workspace/payara-community-documentation"
+
+rm $OUTPUT_NAV_LOCATION
+touch $OUTPUT_NAV_LOCATION
+
+cd $WORKING_DIR
+
+create_nav () {
```

Comment:
Also, some additional recommendations:

* The README pages should be at the top of every section. Personally, I recommend renaming these files to something more simple like "overview"
* The script should handle some custom sorting, possibly based on metadata for each corresponding page. I encourage you to check if Antora handles some standard attributes for this.

Reply:
Version sorting I will implement in the next iteration of the script, there is some more sorting tweaks I would like to do also. Otherwise, a sorting system using a 'ordinal' metadata tag in asciidoc determines the order of the documentation, all 'Overview' pages have a ordinal of 900 and will be at the top of every section. Files without the tag are sorted in whatever order they are in the file structure.